### ISSUE TITLE
v1.0.0.beta5 release not available via rubygems.org

### ISSUE THREAD

**Issue by MrBananaLord** [ahx/openapi_first#196]
Hi :) I (and a couple of other people) have an issue with installing the newest version (beta5) from rubygems.
I know it's probably not a code-related issue (I tested the gem directly from the main branch and it works fine), but I couldn't find a better place to raise it.

```
# when using gem install
gem install openapi_first --pre --version "1.0.0.beta5"
ERROR:  Could not find a valid gem 'openapi_first' (= 1.0.0.beta5) in any repository
ERROR:  Possible alternatives: openapi_first

# when setting version in the Gemfile
Your bundle is locked to openapi_first (1.0.0.beta5) from rubygems repository https://rubygems.org/ or installed
locally, but that version can no longer be found in that source. That means the author of openapi_first (1.0.0.beta5)
has removed it. You'll need to update your bundle to a version other than openapi_first (1.0.0.beta5) that hasn't been
removed in order to install.
```

At the same time I can successfully download the gem from the rubygems https://rubygems.org/gems/openapi_first/versions/1.0.0.beta5 🤔

**Comment by ahx** [comment]
I currently have the same problem, but I got the notification email from rubygems.org that the version (1.0.0.beta5) was pushed to rubygems. So I guess there is nothing else I can do here except wait a little while(?).

**Comment by ahx** [comment]
This seems to be the same issue as described in https://github.com/rubygems/rubygems.org/issues/4097

**Comment by MrBananaLord** [comment]
Yeah looks exactly like that issue. The beta5 is missing from `curl https://index.rubygems.org/info/openapi_first` while being present in the `curl https://rubygems.org/api/v1/versions/openapi_first.json`

I wonder if pushing a new version (e.g. `1.0.0.beta6`) would do the trick. It would probably get that info file updated :D Kinda a cheesy option, but might work (I expect it to be a cache invalidation issue there so we could also just wait and see)

**Comment by ahx** [comment]
@MrBananaLord Can you try out the beta release by loading the gem from github? `branch: 'v1.0.0.beta5'`
If that is not an option for you I can publish a new version later today.

**Comment by MrBananaLord** [comment]
That's what I am doing and it works great :) (using a tag tho because the branch is not there `gem "openapi_first", github: "ahx/openapi_first", tag: "v1.0.0.beta5"`)

**Comment by ykt** [comment]
Hi @ahx 

> If that is not an option for you I can publish a new version later today.

I bumped same issue, it would be great if you could bump the newer version

**Comment by ahx** [comment]
@ykt I have just released [1.0.0.beta6](https://github.com/ahx/openapi_first/blob/main/CHANGELOG.md#100beta6)

Installing that version seems to work fine so I am closing this. Thanks for the report!
